docs/esi_introduction.md

```diff
@@ -55,5 +55,7 @@ There are no rate limits in place, ESI relies on caching more than rate limiting
 
 ESI returns standard caching headers if the data is cached. Applications should notice and make use of these headers (expires and last-modified) on routes where they are provided.
 
+Some endpoints, specifically for sending mail and reading contracts, have internal rate limits enforced by the EVE monolith. If these rate limits are exceeded, ESI will return HTTP status code 520. 
```

Comment:
Christ it took me some time to hunt this down and verify it.

Are these 520 errors counted towards a users error rate limit?

Reply:
If you mean global per IP error limit - yes.